controllers/product.js

```diff
@@ -0,0 +1,201 @@
+const path = require('path');
+const fs = require('fs');
+const crypto = require('crypto');
+const { EventEmitter } = require('events');
+const { BadRequest } = require('../errorList');
+const { setImagesList, setVideosList } = require('../services/media.service');
+
+const eventEmitter = new EventEmitter();
+const productsList = 'products.store.json';
+
+const now = new Date();
+const today = now.toUTCString().slice(5, 25);
+
+eventEmitter.on('fileUploadFailed', () => {
+  const text = `${today} - Error occurred, file upload was failed`;
+  const jsonText = JSON.stringify(text);
+
+  fs.appendFile('filesUpload.log', jsonText + '\n', (err) => {
```

Comment:
create a separate function and move writing to the log file there in order to avoid code duplication